tests/pkcs1v15.rs

```diff
@@ -21,3 +25,55 @@ fn signature_stringify() {
     assert_eq!(format!("{:X}", signature), expected);
     assert_eq!(signature.to_string(), expected);
 }
+
+#[test]
```

Comment:
To use PEM here you'll have to gate this on the `pem` feature as above:

```suggestion
#[cfg(feature = "pem")]
#[test]
```